rfc/rfc002-authentication-token.md

```diff
@@ -401,3 +401,129 @@ Beside the mandatory VP fields, the following applies:
 * The `proof.type` field MUST equal `NutsUziSignedContract`.
 * The `proof.proofValue` field MUST contain the JWT in its compact serialization form as described in \[[RFC7515 section 3.1](https://tools.ietf.org/html/rfc7515#section-3.1)\].
 
+### 7.3 NutsEmployeeIdentity
+
+The NutsEmployeeIdentity method is an authentication _means_ which allows a care organisation to assert the identity of the current user.
+The care organisation uses the information of the current user session to present the user with a statement about its identity and asks the user to confirm this.
+After confirmation this bundle of information is singed with the care organisation's private key.
+
+This method makes it possible to assert a user identity without the need for a personal authentication means.
```

Comment:
with lower level of assurance (LoA). Now it's only a note, but it's a very important trait of this authentication means.